app/assets/stylesheets/basis/_base.scss

```diff
@@ -3,32 +3,103 @@
 }
 
 body {
-  font-size: 24px;
-  font-family: "Sentinel", "helvetica";
+  font-size: 23px;
+  font-family: "Tiempos", "helvetica";
```

Comment:
I'll show you how it's like on heroku. 22px seemed too little, 24px too much ahha. We can change that in another PR later too.